src/settings/SelectMailAddressForm.ts

```diff
@@ -68,6 +68,7 @@ export class SelectMailAddressForm implements Component<SelectMailAddressFormAtt
 			label: "mailAddress_label",
 			value: this.username,
 			alignRight: true,
+			autocompleteAs: Autocomplete.newPassword,
```

Comment:
The same line should also be added to SelectMailAddressFormWithSuggestions since this one is used to show suggestions